crypto/thread/arch/thread_posix.c

```diff
@@ -193,6 +193,7 @@ void ossl_crypto_condvar_free(CRYPTO_CONDVAR **cv)
     *cv_p = NULL;
 }
 
+# if 0                           /* We don't use this anywhere */
```

Comment:
Delete instead of `#if 0` it out.

Reply:
I will, if we determine that this is something that we'll never use.